activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/ProcessVariablesPayloadConverter.java

```diff
@@ -56,6 +56,8 @@ public StartProcessPayload convert(StartProcessPayload payload) {
                     .withName(payload.getName())
                     .withProcessDefinitionId(payload.getProcessDefinitionId())
                     .withProcessDefinitionKey(payload.getProcessDefinitionKey())
+                    .withLinkedProcessInstanceId(payload.getLinkedProcessInstanceId())
+                    .withLinkedProcessInstanceType(payload.getLinkedProcessInstanceType())
```

Comment:
Please, update ProcessVariablesPayloadConverterTest to add coverage for these lines